### PR TITLE
PS-2866 Add project.mlflowAbsConnectionString

### PR DESCRIPTION
**Pull request by joker806**
https://keboola.atlassian.net/browse/PS-2866
Depends on https://github.com/keboola/sandboxes-api/pull/151

### CODE REVIEW

File: src/Project.php [hunk 1]
```diff
@@ -9,18 +9,20 @@ class Project
...
-        $project->setMLflowAbsSas($in['mlflowAbsSas'] ?? '');
-        $project->setCreatedTimestamp($in['createdTimestamp']);
-        $project->setUpdatedTimestamp($in['updatedTimestamp'] ?? '');
+        $project = new self();
+        $project->id = (string) $in['id'];
+        $project->mlflowUri = $in['mlflowUri'] ?? '';
+        $project->mlflowAbsSas = $in['mlflowAbsSas'] ?? '';
+        $project->mlflowAbsConnectionString = $in['mlflowAbsConnectionString'] ?? '';
+        $project->createdTimestamp = $in['createdTimestamp'];
+        $project->updatedTimestamp = $in['updatedTimestamp'] ?? '';
```
Comment (pivnicek) [inline src/Project.php:25]: just curious why this change from setters?  (though I must say I prefer `new self()`)

**Review by pivnicek — APPROVED** [review]
great!  What about just adding a new assertion to this test? https://github.com/keboola/sandboxes-api-php-client/blob/master/tests/ClientTest.php#L201

File: src/Project.php (hunk 1 above)
Comment (joker806) [reply src/Project.php:25]: There is no setter for `mlflowAbsConnectionString` (it's generated on the API server side) and I wantend to be consitent.

Also it's not nessecary, the method is part of the class, so it has access to its private scope.

**Comment by joker806** [comment]
Jo, diky za ty testy, nejak mi to tu uteklo. Doplneno..
